pkg/purl/purl.go

```diff
@@ -0,0 +1,111 @@
+package purl
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/aquasecurity/fanal/analyzer"
+	"github.com/aquasecurity/fanal/analyzer/os"
+	"github.com/aquasecurity/fanal/types"
+
+	"github.com/package-url/packageurl-go"
+)
+
+func NewPackageURL(t string, pkg types.Package) packageurl.PackageURL {
+	name := strings.ReplaceAll(pkg.Name, ":", "/")
```

Comment:
could you add test case for the replacement?

Reply:
thx. I fixed